HLTrigger/Configuration/python/HLT_75e33/psets/TrackAssociatorParameters_cfi.py

```diff
@@ -1,15 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
 TrackAssociatorParameters = cms.PSet(
-    CSCSegmentCollectionLabel = cms.InputTag("cscSegments"),
+    CSCSegmentCollectionLabel = cms.InputTag("hltCscSegments"),
     CaloTowerCollectionLabel = cms.InputTag("towerMaker"),
     DTRecSegment4DCollectionLabel = cms.InputTag("dt4DSegments"),
```

Comment:
Should this be changed to `hltDt4DSegments` ?

Reply:
This question about `TrackAssociatorParameters.DTRecSegment4DCollectionLabel` still applies.